docs/raising-events.md

```diff
@@ -95,17 +96,33 @@ forwarded to another implementation (wrapped object or base class) that
 FakeItEasy has no control over, so the fake doesn't know about the handlers and
 cannot call them.
 
-To work around this limitation, you will need to explicitly configure the calls
-to the event accessors to handle event subscription yourself, as in the example
-below:
+Similarly, strict fakes don't handle any call unless explicitly configured,
+including event subscription or unsubscription, so FakeItEasy also can't raise
+events on strict fakes.
+
+To work around this limitation, you have three options:
+
+- For a strict fake, you can enable the default event behavior on the fake at
+creation time, as explained [here](strict-fakes.md#events).
+
+- You can explicitly enable the default event behavior on the fake, for a
+specific event or for all events of the fake:
+
+```

```

Comment:
```suggestion
```csharp
```

Otherwise we see 
![image](https://user-images.githubusercontent.com/3275797/120070293-bcf82980-c057-11eb-81a3-fe8039c6162b.png)
instead of 
![image](https://user-images.githubusercontent.com/3275797/120070435-8a9afc00-c058-11eb-9827-1cd72d73c272.png)


(and the highlighting is nicer in vscode :wink:)